### PR TITLE
netsuite password fix

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Review by ashwin1111 — APPROVED** [review]

File: src/app/shared/components/configuration/configuration-connector/configuration-connector.component.html [hunk 1]
```diff
@@ -40,13 +40,13 @@ <h3 *ngIf="!accountingCompanyConnectionInProgress && accountingCompanyName" clas
...
                 </div>
-                <div class="tw-w-300-px tw-text-pretty tw-text-center tw-py-16-px">
+                <div class="tw-w-300-px tw-text-pretty tw-text-center tw-py-16-px" *ngIf="!accountingCompanyConnectionInProgress">
                     <span class="tw-text-14-px tw-text-normal-text-color tw-font-500 tw-text-center">{{subLabel}}</span>
                 </div>
             </div>
         </div>
         <div *ngIf="accountingCompanyConnectionInProgress" class="tw-pb-8-px tw-text-center">
-            <p class="tw-text-14-px tw-pb-16-px tw-w-300-px tw-text-text-muted tw-text-center" *ngIf="brandingConfig.brandId === 'co'">
+            <p class="tw-text-14-px tw-py-16-px tw-w-300-px tw-text-text-muted tw-text-center" *ngIf="brandingConfig.brandId === 'co'">
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-connector/configuration-connector.component.html:49]: test qbo for this change if there is no extra space that got added, pls also add a screen recording

File: src/app/shared/components/configuration/configuration-connector/configuration-connector.component.html [hunk 2]
```diff
@@ -19,7 +19,7 @@ <h5 class="tw-text-14-px tw-text-text-tertiary tw-font-500 tw-pb-18-px tw-text-c
         <h3 *ngIf="!accountingCompanyConnectionInProgress && accountingCompanyName" class="tw-text-18-px tw-text-text-primary tw-font-500 tw-pb-16-px tw-text-center ">
             {{ accountingCompanyName }}
         </h3>
-        <div *ngIf="!accountingCompanyConnectionInProgress && !accountingCompanyName && !isDisconnectClicked" class="tw-flex tw-items-center tw-justify-center">
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-connector/configuration-connector.component.html:22]: pls test business central and add a screen recording

File: src/app/shared/components/netsuite/core/netsuite-subsidiary-mapping/netsuite-subsidiary-mapping.component.ts [hunk 3]
```diff
@@ -87,6 +86,7 @@ export class NetsuiteSubsidiaryMappingComponent implements OnInit {
         this.netsuiteSubsidiary = netsuiteSubsidiary;
         this.isLoading = false;
         this.handleSuccess(netsuiteSubsidiaryMappingPayload);
+        // This.saveInProgress = false;
```
Comment (ashwin1111) [inline src/app/shared/components/netsuite/core/netsuite-subsidiary-mapping/netsuite-subsidiary-mapping.component.ts:89]: remove it off

**Review by ashwin1111 — APPROVED** [review]

**Comment by DhaaraniCIT** [comment]
https://github.com/fylein/fyle-integrations-app/assets/55541808/6c6b8e39-4659-45a4-9c3c-0bcf6a98b80d


https://github.com/fylein/fyle-integrations-app/assets/55541808/f1ce9ded-82f3-46e2-8e94-10558451891d


https://github.com/fylein/fyle-integrations-app/assets/55541808/47503804-8077-47ca-89d2-3e3566d6df80

**Pull request merged**